Symfony/CS/Fixer/PSR2/VisibilityFixer.php

```diff
@@ -61,9 +61,9 @@ public function getDescription()
      *
      * Token at given index is prepended by attributes.
      *
-     * @param Tokens $tokens  Tokens collection
-     * @param int    $index   token index
-     * @param array  $attribs array of token attributes
+     * @param Tokens                    $tokens  Tokens collection
+     * @param int                       $index   token index
+     * @param array<string, Token|null> $attribs array of token attributes
```

Comment:
Are you sure this is supported by all IDEs?

`string[]` and `Token[]` looks like to be more used and parsed.